tls/test/test_hello_message.py

```diff
@@ -139,6 +139,23 @@ class TestClientHello(object):
         b'\x00\x04'
     ) + truncated_hmac_ext_packet
 
+    cert_status_request_ext = (
```

Comment:
oops, added this a day after the above comment, and forgot to update the comment.